frontend/src/pages/GraphPF/components/node.tsx

```diff
@@ -303,11 +314,25 @@ const BaseNodeComponent: React.FunctionComponent<BaseNodeProps> = ({
   }, [element, nodeScale, scaleNode]);
 
   const ColorFind = PFColors.Gold400;
+  const ColorFocus = PFColors.Blue400;
   const ColorSpan = PFColors.Purple200;
   const OverlayOpacity = 0.3;
   const OverlayWidth = 40;
   const UnhighlightOpacity = 0.1;
 
+  const flashAnimation = keyframes({
```

Comment:
```suggestion
  const focusAnimation = keyframes({
```

Just a suggestion, not required to change.